### PR TITLE
Update Android Version to 2.7.0

### PR DESCRIPTION
**Pull request by ambarwootric**
From PR https://github.com/Wootric/WootricSDK-Android/pull/57

### CODE REVIEW

File: source/includes/_install_android.md [hunk 1]
```diff
@@ -5,15 +5,15 @@
 <dependency>
     <groupId>com.wootric</groupId>
     <artifactId>wootric-sdk-android</artifactId>
-    <version>2.6.1</version>
+    <version>2.7.0</version>
```
Comment (diegoserranoa) [inline source/includes/_install_android.md:8]: `2.7.1` or `2.7.0`?

Comment (ambarwootric) [reply source/includes/_install_android.md:8]: It's `2.7.0` @diegoserranoa

Comment (diegoserranoa) [reply source/includes/_install_android.md:8]: Ok, got mislead by the PR title. If you can fix it :)

Comment (ambarwootric) [reply source/includes/_install_android.md:8]: Oops looks like I did write that heading wrong 😠 - Sorry!

**Pull request merged**